aws-cms-oit-iusg-spe-cmcs-macbis-test/ecr.tf

```diff
@@ -1,7 +1,7 @@
 module "cross_account_ecr" {
   source = "../cross-account-ecr"
 
-  account_id = "037370603820"
+  principal_arns = ["arn:aws:iam::037370603820:root", "arn:aws:iam::741306476019:root"] // macbis-dev, MACFin
```

Comment:
You could put the comment against each arn separately:
`["arn:aws:iam::037370603820:root", # macbis-dev
  "arn:aws:iam::741306476019:root", # MACFin
]`

I chose `#` since it is the idiomatic style for single-line comments.